src/mod/externals/Dpr/UI/SettingMenuItem.h

```diff
@@ -26,18 +36,26 @@ namespace Dpr::UI {
             Dpr::UI::UIText::Object* _sliderText;
         };
 
+
         inline void ctor() {
             external<void>(0x01d3fed0, this);
         }
 
-        inline void Setup(int32_t configId, int32_t selectIndex, System::String* descMessageLabel, void* onValueChanged) {
+        inline void Setup(int32_t configId, int32_t selectIndex, System::String* descMessageLabel, UnityEngine::Events::UnityAction::Object* onValueChanged) {
             external<void>(0x01d3ee30, this, configId, selectIndex, descMessageLabel, onValueChanged);
         }
     };
 }
 
 namespace System::Collections::Generic {
     struct List$$SettingMenuItem : List<List$$SettingMenuItem, Dpr::UI::SettingMenuItem> {
+        struct Fields {
```

Comment:
Is there a specific reason you added the Fields here? Normally it should be inheriting them from the generic List